lib/matplotlib/tests/test_widgets.py

```diff
@@ -974,6 +974,24 @@ def onselect(vmin, vmax):
     assert tool.extents == (17, 35)
 
 
+def test_span_selector_extents(ax):
+    def onselect(vmin, vmax):
+        ax._got_onselect = True
```

Comment:
This is never called, according to code coverage. I also don't know why it should be setting an attribute on the `Axes`. Should it instead be raising an error, since `ignore_event_outside` means it should not be called?

Reply:
This is not necessary in this test (I reuse some other tests) so I changed it to a function doing nothing.